src/sentry/static/sentry/app/components/charts/chartZoom.jsx

```diff
@@ -214,10 +214,12 @@ class ChartZoom extends React.Component {
       return children(props);
     }
 
-    const dateFormat = useShortDate ? 'MMM Do' : 'lll';
+    const dateFormat = useShortDate ? 'MMM Do' : 'MMM D LT';
     const hasShortInterval = useShortInterval(this.props);
     const xAxisOptions = {
       axisLabel: {
+        showMaxLabel: false,
```

Comment:
@doralchan this should hide the first and last x-axis labels which was the cause of the clipping.